README.md

```diff
@@ -17,9 +17,11 @@ See other branches for other framework versions:
 * 11.0 for Vaadin 13 (End-of-line)
 * 12.0 for Vaadin 14.0
 * 12.1 for Vaadin 14.1
-* 12.2 for upcoming Vaadin 14.2
+* 12.2 for Vaadin 14.2
+* 12.3 for Vaadin 14.3
```

Comment:
did we have a specific version for 14.4?

Reply:
12.3 also for 14.4. Updated.